csv_handler_spec.rb

```diff
@@ -3,32 +3,16 @@
 
 #describing behaviour of CsvHandler
 describe CsvHandler do
-	subject { CsvHandler.new('data.csv') }
+	subject { CsvHandler.new(file_name) }
+  let(:file_name) { 'data.csv' }
 
   describe '#format' do
-    it 'returns array template' do
-
-      expect(subject.format).to eq([["derek trotter","sa1 1nw", "no"],["Rodney trotter","Sa6 W23","yes"],["Rick Grimes","Sa2 6tr","no"]])    
+    it 'returns contents of csv file in 2d array format' do
```

Comment:
As the assertion already specifies the result, we can use more human friendly language in context and it descriptions

```suggestion
    it 'formats the CSV correctly' do
```